src/.eslintrc.js

```diff
@@ -21,14 +24,7 @@ module.exports = {
     "prefer-import/prefer-import-over-require": [1],
     "no-console": ["warn", { allow: ["assert"] }],
     "no-debugger": 1,
-    "no-unused-vars": 1,
     "arrow-body-style": 1,
-    "valid-jsdoc": [
```

Comment:
The docs for this plugin say that it is deprecated, and to favor `eslint-plugin-jsdoc` https://eslint.org/docs/latest/rules/valid-jsdoc

I think that's why the `requireReturnDescription` rule was not working correctly